### PR TITLE
Bitrise step eschecker 0.0.2

### PR DESCRIPTION
**Pull request by Treyone**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/bitrise-step-eschecker/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+title: esChecker, by eShard
```
Comment (ofalvai) [inline steps/bitrise-step-eschecker/0.0.1/step.yml:1]:
```suggestion
title: esChecker by eShard
```

File: steps/bitrise-step-eschecker/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+toolkit:
+  bash:
+    entry_file: step.sh
+inputs:
+- opts:
+    description: |
+      Path to the binary file to be tested
+    is_required: true
+    summary: The app file you want to test.
+    title: A Bitrise generated APK or IPA path
```
Comment (ofalvai) [inline steps/bitrise-step-eschecker/0.0.1/step.yml:24]: Just out of curiosity, does this accept an Android AAB too?

File: steps/bitrise-step-eschecker/0.0.2/step.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    title: Application ID
+- eschecker_campaign_id: null
+  opts:
+    description: |
+      Campaign ID in esChecker
+    is_required: true
+    summary: Unique ID of the campaign you can to run in esChecker
+    title: Campaign ID
+outputs:
+- eschecker_pdf_report: null
```
Comment (ofalvai) [inline steps/bitrise-step-eschecker/0.0.2/step.yml:50]: If it's a local path, I would rather call this output `eschecker_report_path`, making it consistent with other steps that provide path outputs.

File: steps/bitrise-step-eschecker/0.0.1/step.yml (hunk 2 above)
Comment (Treyone) [reply steps/bitrise-step-eschecker/0.0.1/step.yml:24]: No, not yet. APK only for now.

**Pull request closed without merging**